worlds/civ_6/Options.py

```diff
@@ -0,0 +1,115 @@
+from dataclasses import dataclass
+from Options import Choice, DeathLink, DefaultOnToggle, PerGameCommonOptions, Range, StartInventory, Toggle
+
+
+class ProgressionStyle(Choice):
+    """Determines what progressive items (if any) should be included.\n\n
+    Districts Only: Each tech/civic that would normally unlock a district or district building now has a logical progression. Example: TECH_BRONZE_WORKING is now PROGRESSIVE_ENCAMPMENT\n\n
+    Eras and Districts: Players will be defeated if they play until the world era advances beyond the currently unlocked maximum era.
+    A notification will be shown as the end of the era approaches letting the player know if they don't have enough progressive era items.
+    Unlocked eras can be seen in both the tech and civic trees. Includes all progressive districts.\n\n
+    None: No progressive items will be included. This means you can get district upgrades that won't be usable until the relevant district is unlocked.
+    """
+    display_name = "Progression Style"
+    option_districts_only = 0
+    option_eras_and_districts = 1
+    option_none = 2
+    default = option_districts_only
+
+
+class ShuffleGoodyHuts(DefaultOnToggle):
+    """Shuffles the goody hut rewards. Goody huts will only contain junk items and location checks are received sequentially (GOODY_HUT_1, GOODY_HUT_2, etc)."""
+    display_name = "Shuffle Goody Hut Rewards"
+
+
+class BoostSanity(Toggle):
+    """Boosts for Civics/Techs are location checks. Boosts can now be triggered even if the item has already been researched. If it is dependent upon a unit that is now obsolete, you can click toggle on/off the relevant tech in the tech tree."""
+    display_name = "Boostsanity"
+
+
+class ExcludeMissableBoosts(Toggle):
+    """If boostsanity is enabled, this will prevent any boosts that are 'missable' from having progression items. Disabling this will potentially require multiple playthroughs to complete the seed."""
+    display_name = "Exclude Missable Boosts"
+
+
+class ResearchCostMultiplier(Choice):
+    """Multiplier for research cost of techs and civics, higher values make research more expensive. Cheap = 0.5x, Expensive = 1.5x. Default is 1. """
+    display_name = "Tech/Civic Cost Multiplier"
+    option_cheap = 0.5
+    option_default = 1
+    option_expensive = 1.5
+    default = 1
+
+
+class PreHintItems(Choice):
+    """Controls if/what items in the tech/civics trees are pre hinted for the multiworld.\n
+    All : All items in the tech & civics trees are pre hinted.\n
```

Comment:
```suggestion
    All: All items in the tech & civics trees are pre-hinted.\n
```
Shouldn't be a space before the colon.
Also, I _think_ it's more proper to do pre-hint instead of pre hint